code/modules/holodeck/computer.dm

```diff
@@ -25,44 +31,54 @@
 	idle_power_usage = 10
 	active_power_usage = 50
 
-	var/area/holodeck/linked
-	var/area/holodeck/program
-	var/area/holodeck/last_program
-	var/area/offline_program = /area/holodeck/rec_center/offline
+	var/area/holodeck/linked //the area that this holodeck loads templates into, used for power and deleting holo objects that leave it
+
+	var/area/mappedstartarea = /area/holodeck/rec_center //change this to a different area if youre making a second holodeck different from the station one
+	var/program = "holodeck_offline" //what program is loaded right now
+	var/last_program
 
-	var/list/program_cache
-	var/list/emag_programs
+	var/offline_program = "holodeck_offline" //the default program loaded by this holodeck when spawned and when deactivated
+	var/holodeck_access = STATION_HOLODECK //what access type this holodeck has, used to specify programs for another holodeck that others cant load
```

Comment:
Could you document new variables?

Reply:
where would you want me to document them?

Reply:
done